pkg/test/ginkgo/synthentic_tests.go

```diff
@@ -84,10 +90,44 @@ func createSyntheticTestsFromMonitor(m *monitor.Monitor, eventsForTests []*monit
 	syntheticTestResults = append(syntheticTestResults, testPodTransitions(events)...)
 	syntheticTestResults = append(syntheticTestResults, testSystemDTimeout(events)...)
 	syntheticTestResults = append(syntheticTestResults, testPodSandboxCreation(events)...)
+	syntheticTestResults = append(syntheticTestResults, testServerAvailability("kube-apiserver", events, monitorDuration)...)
```

Comment:
If I'm reading this correctly, the `kube-apiserver` is a well-known name somewhere that is used solely to indicate availability?

Reply:
Added consts to `monitor` pkg.